101-load-balancer-ip-configured-backend-pool/README.md

```diff
@@ -2,14 +2,10 @@
 ![Azure Public Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/101-load-balancer-ip-configured-backend-pool/PublicLastTestDate.svg)
 ![Azure Public Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/101-load-balancer-ip-configured-backend-pool/PublicDeployment.svg)
 
-![Azure US Gov Last Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/101-load-balancer-ip-configured-backend-pool/FairfaxLastTestDate.svg)
```

Comment:
the badges still need to be present, to show that FF is not supported.

Reply:
updated